aspnetcore/signalr/version-differences.md

```diff
@@ -43,15 +53,15 @@ app.UseSignalR(routes =>
 
 ### Sticky sessions now required
 
-Because of how scale-out worked in the previous versions of SignalR, clients could reconnect and send messages to any server in the farm. Due to changes to the scale-out model, as well as not supporting reconnects, this is no longer supported. Now, once the client connects to the server it needs to interact with the same server for the duration of the connection.
+Because of how scale-out worked in ASP.NET SignalR, clients could reconnect and send messages to any server in the farm. Due to changes to the scale-out model, as well as not supporting reconnects, this is no longer supported. Now, once the client connects to the server it needs to interact with the same server for the duration of the connection.
```

Comment:
Comma splice (out of position) ... You can use "Now" or not if you prefer (the style manual slightly dissuades it), but structure either of these ways (comma after "server") ...

* Now once the client connects to the server, it needs to interact ...
* Once the client connects to the server, it needs to interact ...

Recommend "requires" over "needs," since "needs" personifies inanimate objects.

Reply:
"requires" is a bit clunky here, going with "it must interact with the same server...", but feel free to smith that further :)

Reply:
Possibly could just drop it ...

> Once the client connects to the server, it interacts with the same server for the duration of the connection.

Reply:
That looks good ...

> ... it must interact with the same server ...